Tools/Cli-LoRa-Device-Provisioning/Cli-LoRa-Device-Provisioning.csproj

```diff
@@ -5,9 +5,6 @@
     <TargetFramework>$(TargetFramework)</TargetFramework>
     <RootNamespace>LoRaWan.Tools.CLI</RootNamespace>
     <AssemblyName>loradeviceprovisioning</AssemblyName>
-    <CodeAnalysisTreatWarningsAsErrors>false</CodeAnalysisTreatWarningsAsErrors>
-    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <EnforceCodeStyleInBuild>false</EnforceCodeStyleInBuild>
```

Comment:
If not mistaken this [is disabled](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#enforcecodestyleinbuild) by default, don't we want to enable it?

Reply:
This is automatically picked up from the `Directory.Build.props` that we have in the solution.